etl/steps/data/garden/who/2025-04-07/gho_smoking.meta.yml

```diff
@@ -36,6 +31,9 @@ tables:
         presentation:
           grapher_config:
             note: "{definitions.notes.age_standardization}"
+        description_key:
```

Comment:
The data page for some of these variables still has a bit of a machine-y title because of the sex dimension, it would be good to have these a bit nicer (e.g. [here](http://staging-site-enhance-improve-who-empower/admin/datapage-preview/1025630))

Reply:
Would be good to alter the unit as well, based on if it's % of men, or % of women etc